### ISSUE TITLE
Not all SC have comments in Appendix A - is this deliberate?

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#1107]
Appendix A has 36 items, but there are 50 WCAG 2.1 SC at AA. I see that the listing for 1.2 has five that are listed in one, so that makes 40. I see that some are missing such as 4.1.1 and 4.1.3, but am not sure if this is intentional.

Also, there is no Success Criterion 1.2, it should say Guideline 1.2

**Comment by peterkorn** [comment]
@awkawk - Yes. Quoting the second sentence of Appendix A: "In this draft, the list is not exhaustive, but we intend it to cover all known challenges with all A and AA Success Criteria, by the time this Note is completed."

We felt there was enough in Appendix A to be sufficient for FPWD.

**Comment by awkawk** [comment]
Ah, got it. I missed that sentence, and the list was long enough to seem like it was trying to be comprehensive.

The second comment remains as an editorial detail.

**Comment by JaninaSajka** [comment]
Second comment now fixed in current PR (#1109)
